src/components/ObjectInput.tsx

```diff
@@ -0,0 +1,103 @@
+/* @jsx jsx */
+import { jsx } from '@emotion/core';
+import * as React from 'react';
+
+import { Box, Button, Flex, Text } from '@stoplight/ui-kit';
+
+import { fieldName, IFormtronControl } from '..';
+import { DraftValue } from './DraftValue';
+import { ValidityIndicator } from './ValidityIndicator';
+
+export const ObjectInput: React.FunctionComponent<IFormtronControl> = ({
+  id,
+  value = {},
+  schema,
+  onChange,
+  fieldComponents,
+  selection,
+}) => {
+  // Make shallow copy
+  const items = [...Object.entries(value)];
+  const defaultValue = schema.default;
+  const KeyWidget = fieldComponents[fieldName(schema.keys)];
+  const ValWidget = fieldComponents[fieldName(schema.values)];
+
+  const fromEntries = (entries: Array<[string, {}]>) => {
```

Comment:
Um... how about `lodash.fromPairs`?

(Ideally, we would use `Object.fromEntries`, but as we discussed earlier, we'd need to add a polyfill for Jest and Studio)

Reply:
Bahaha, I got bogged down reading https://github.com/tc39/proposal-object-values-entries/issues/15 and never even realized there was (later) a `fromEntries` proposal.

yeah `fromPairs` is perfect.